source/manage-aspas.rst

```diff
@@ -4,20 +4,23 @@ Manage ASPA Objects
 ===================
 
 .. Warning:: The IETF is still discussing and recently changed the ASPA
-             object profile. Krill currently supports the previous (V0)
-             version of the profile. RPKI validators may reject these
-             objects.
+             object profile. Krill currently supports the latest version
+             of the profile.
 
-             The change to be aware of is that the new version dropped
-             the optional AFI limit for provider ASNs. Krill 0.14.0 will
-             support the new profile. Any existing configurations will be
-             re-imported on upgrade (excluding the AFI limit). This
-             migration will remove the history for pre-0.14.0 ASPA
-             configurations so that we can remove the AFI limit concept
-             from the code, thus improving future maintainability.
+             Any existing ASPA objects that you created before Krill
+             0.14.0 will be recreated on upgrade. If you used the now
+             deprecated option to limit a provider to a certain AFI,
+             then that limit will be silently dropped on upgrade.
 
-             In short, you can experiment with ASPA objects in Krill, but
-             keep the above in mind.
+             Also note that ASPA objects are only supported in the API
+             and CLI for the moment. The API changed in 0.14.0 and will
+             now use simple numbers for the ASNs in JSON. The CLI still
+             uses the more human friendly string notation which allows
+             both simple numners, e.g. "65000", or "AS65000" to be used.
+
+             We plan to add support for ASPA management in the UI in
+             future, probably shortly after the ASPA profile will have
+             made through IETF last call.
```

Comment:
"IETF last call" maybe a bit too jargony for people not familiar?

Reply:
fixed